package.json

```diff
@@ -106,6 +106,7 @@
         "bignumber.js": "^9.0.1",
         "bn.js": "^5.2.1",
         "browserify": "^16.5.2",
+        "browserify-zlib": "^0.2.0",
```

Comment:
why this lib is added?

Reply:
we need this Polyfill otherwise build fails by [error](https://github.com/web3/web3.js/actions/runs/5548744411/jobs/10132070027)